### PR TITLE
Fist iteration: add pics?

### PR DESCRIPTION
**Pull request by TatianaLekhatkova**
Added description. Add more pics? Make it more funny?

### CODE REVIEW

File: docs/guides/use-team-admin-notifications.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+Send notification, when new event is created from your Vyte page. As an owner of Vyte page you are always considered as an event creator.
+
+</attribute>
+
+## Set up your team admin preferences
+To set up your team admin preferences you can use the `/teams/:team_id/admin-preferences` endpoint with your team `_id` and POST method. If some preferences are not defined they will be set up `false` par default.
+
+```shell screen-hidden
+curl --request POST 'https://api.vyte.in/teams/:team_id/admin-preferences' \
```
Comment (saintmac) [inline docs/guides/use-team-admin-notifications.md:92]: always show /v2/ endpoints in the documentation

File: docs/guides/use-team-admin-notifications.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+--header 'Authorization: 2lnpjjrurrl49xja5oo0qujtl60embr7zppiphc5fcav4n7ycx' \
```
Comment (saintmac) [inline docs/guides/use-team-admin-notifications.md:93]: just to be sure is it a fake one ?

**Review by saintmac — CHANGES_REQUESTED** [review]
Explain how teams work and how to create them first. Unless we already have that in another guide that you can reference.

use /v2/ endpoint.

File: docs/guides/use-team-admin-notifications.md (hunk 1 above)
Comment (TatianaLekhatkova) [reply docs/guides/use-team-admin-notifications.md]: done 5cbfb535239a5ff71cbe94684857a838d764f48b

File: docs/guides/use-team-admin-notifications.md (hunk 2 above)
Comment (TatianaLekhatkova) [reply docs/guides/use-team-admin-notifications.md]: absolutely fake, corrected a bit: 5cbfb535239a5ff71cbe94684857a838d764f48b

**Review by saintmac — APPROVED** [review]

**Pull request merged**